src/controllers/order/create.ts

```diff
@@ -56,7 +56,7 @@ const create = async (req: BodyRequest<Body>, res: Response) => {
       })
       .reduce(
         (acc, item) => {
-          if (item.item.category.key !== 'pizzas') {
+          if (item.item.category.needsPreparation === true) {
```

Comment:
```suggestion
          if (item.item.category.needsPreparation) {
```
Vu que c'est un booléen t'es pas obligé de préciser égal à true ?